tests/test_benchmark_factory.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+from common.benchmark_factory import BenchmarkFactory
+from common.benchmark_wrapper import BenchmarkWrapper
+
+
+class PermuteArguments(unittest.TestCase):
```

Comment:
I don't really understand the class name! I Would go with some "TestArguments" instead of "PermuteArguments".

Reply:
The above change has been made